README.md

```diff
@@ -21,38 +22,49 @@ Your task is to make them pass, using Eclipse Collections.
 Quick Example
 -------------
 
-Eclipse Collections puts iteration methods on the container types. Lambdas are simulated using anonymous inner classes. Here's a code example that demonstrates the usual style of programming with Eclipse Collections.
+Eclipse Collections puts iteration methods directly on the container types. Here's a code example that demonstrates the simple style of programming with Eclipse Collections.
 
 ```java
-MutableList<Person> people = FastList.newListWith(person1, person2, person3);
-MutableList<String> sortedLastNames = people.collect(Person.TO_LAST_NAME).sortThis();
-System.out.println("Comma separated, sorted last names: " + sortedLastNames.makeString());
+MutableList<Person> people = Lists.mutable.with(person1, person2, person3);
+MutableList<String> lastNames = people.collect(Person::getLastName);
+System.out.println("Comma separated last names: " + lastNames.makeString());
 ```
-
-Person.TO_LAST_NAME is defined as a constant Function in the Person class.
+The method reference in the example above can also be replaced with a lambda:
 
 ```java
-public static final Function<Person, String> TO_LAST_NAME = new Function<Person, String>()
-{
-    public String valueOf(Person person)
-    {
-        return person.lastName;
-    }
-};
+MutableList<String> lastNames = people.collect(person -> person.getLastName());
+```
+Eclipse Collections has support for both Mutable and Immutable collections, and the return types of methods are covariant.
 
+```java
+ImmutableList<Person> people = Lists.immutable.with(person1, person2, person3);
+ImmutableList<String> lastNames = people.collect(Person::getLastName);
+System.out.println("Comma separated last names: " + lastNames.makeString());
 ```
-In Java 8, the Function can be replaced with a lambda:
+Eclipse Collections has a lazy API as well, which is available by calling the method asLazy().
 
 ```java
-MutableList<String> sortedLastNames = people.collect(person -> person.getLastName()).sortThis();
+ImmutableList<Person> people = Lists.immutable.with(person1, person2, person3);
+LazyIterable<String> lastNames = people.asLazy().collect(Person::getLastName);
+System.out.println("Comma separated last names: " + lastNames.makeString());
 ```
+The MutableCollections in Eclipse Collections also have the Stream API available in Java, since they extend their corresponding JDK types.
 
-Or, a method reference:
+```java
+MutableList<Person> people = Lists.mutable.with(person1, person2, person3);
+Stream<String> lastNames = people.stream().map(Person::getLastName);
+System.out.println("Comma separated last names: " + lastNames.collect(Collectors.joining(", "));
+```
+Eclipse Collections 8.0 also introduced a new set of Collectors in the class Collectors2.
```

Comment:
You might consider linking Collectors2 to either the JavaDoc: http://www.eclipse.org/collections/javadoc/8.0.0/org/eclipse/collections/impl/collector/Collectors2.html

or actual file: 
https://github.com/eclipse/eclipse-collections/blob/master/eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/Collectors2.java